subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_fota

```diff
@@ -157,9 +167,13 @@ config NRF_CLOUD_FOTA_POLL
 	  to nRF Cloud.
 
 if NRF_CLOUD_FOTA_POLL
-
 module = NRF_CLOUD_FOTA_POLL
 module-str = nRF Cloud FOTA Poll
 source "subsys/logging/Kconfig.template.log_config"
-
 endif # NRF_CLOUD_FOTA_POLL
+
+if NRF_CLOUD_FOTA_POLL || NRF_CLOUD_FOTA
+module = NRF_CLOUD_FOTA
+module-str = nRF Cloud FOTA
+source "subsys/logging/Kconfig.template.log_config"
```

Comment:
The Kconfig.template.log_config will be "included" regardless of NRF_CLOUD_FOTA_POLL or NRF_CLOUD_FOTA, [because the `if` statement in a Kconfig file does not affect the `source` statement](https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/build/kconfig/tips.html#lack_of_conditional_includes)